dotcom-rendering/eslint-guardian.js

```diff
@@ -17,14 +17,14 @@ const guardianRules = {
 	// use `string[]` for simple arrays, `Array<string>` for complex ones
 	// https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/array-type.md#array-simple
 	'@typescript-eslint/array-type': [
-		WARNING,
+		OFF,
 		{
 			default: 'array-simple',
 		},
 	],
 
 	// use `Record<string, unknown>` instead of `{ [key: string]: unknown }`
-	'@typescript-eslint/consistent-indexed-object-style': [WARNING, 'record'],
+	'@typescript-eslint/consistent-indexed-object-style': [OFF, 'index-signature'],
```

Comment:
So just to check, the rule is being turned off, but we are moving to index-signature?

Reply:
We're _keeping_ index-signature. Once we make a decision we can enforce and auto-fix.